app/models/course_enrichment.rb

```diff
@@ -43,4 +43,9 @@ class CourseEnrichment < ApplicationRecord
   def has_been_published_before?
     last_published_timestamp_utc.present?
   end
+
+  def unpublish(initial_draft: true)
+    update(status: :draft)
+    update(last_published_timestamp_utc: nil) if initial_draft
```

Comment:
Is it worth it to hit the DB twice here?

Reply:
fixed